src/main/java/com/minecolonies/coremod/colony/ColonyManager.java

```diff
@@ -952,7 +968,7 @@ public static void onWorldLoad(@NotNull final World world)
 
                     if (data.hasKey(TAG_NEW_COLONIES))
                     {
-                        final int size = data.getInteger(TAG_NEW_COLONIES);
+                        final int size = data.getInteger(TAG_NEW_COLONIES); // todo Confirm "amountOfColonies" = "max Colony ID"
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this trailing comment on the previous empty line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3ATrailingCommentCheck)
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1135)